src/compat/eme/custom_media_keys/ie11_media_keys.ts

```diff
@@ -145,10 +146,23 @@ export default function getIE11MediaKeysCallbacks() {
     }
     return MSMediaKeysConstructor.isTypeSupported(keySystem);
   };
-  const createCustomMediaKeys = (keyType: string) => new IE11CustomMediaKeys(keyType);
+  const createCustomMediaKeys = (keyType: string) =>
+    new IE11CustomMediaKeys(keyType);
+  const setMediaKeys = (elt: HTMLMediaElement,
+                        mediaKeys: MediaKeys|ICustomMediaKeys|null): void => {
+    if (mediaKeys === null) {
+      return;
+    }
+    if (!isCustomMediaKeys(mediaKeys)) {
+      throw new Error("Custom setMediaKeys not supposed to be called " +
+                      "with IE11 MediaKeys direclty.");
```

Comment:
Quiz: There is a common mistake between the name of your PR and that line (on two different words)...

Reply:
I've never been diagnosed as dyslexic, maybe i should check that 🤔